Source/Scene/GltfLoader.js

```diff
@@ -482,7 +482,8 @@ function loadVertexAttribute(loader, gltf, accessorId, gltfSemantic, draco) {
     gltf,
     accessorId,
     gltfSemantic,
-    draco
+    draco,
+    false
```

Comment:
Explicility passing in `false` here in case the default change were to change in the future or something